### PR TITLE
🐛 Fixed color of member alert email links

### PR DESCRIPTION
**Pull request by lenabaidakova**
closes TryGhost/Team#2015
- Gmail wraps emails in link and added them custom styles. Was added fix to reset Gmail styles.

### CODE REVIEW

File: ghost/staff-service/lib/email-templates/partials/commonStyles.hbs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    }
+    .ExternalClass,
+    .ExternalClass p,
+    .ExternalClass span,
+    .ExternalClass font,
+    .ExternalClass td,
+    .ExternalClass div {
+      line-height: 100%;
+    }
+    /* Reset styles for Gmail (it wraps email address in link with custom styles) */
```
Comment (lenabaidakova) [inline ghost/staff-service/lib/email-templates/partials/commonStyles.hbs:65]: The PR looks large, but in general I just added this comment and add `text-link` class to markup. All others  changes - just moving common styles to the separate file

File: ghost/staff-service/lib/email-templates/partials/commonStyles.hbs [hunk 2]
```diff
@@ -0,0 +1,99 @@
+<style>
```
Comment (rishabhgrg) [inline ghost/staff-service/lib/email-templates/partials/commonStyles.hbs:1]: We don't use camel case filenames, the file name here can be `common-styles.hbs` or just `styles.hbs`

**Review by rshbhgrg — CHANGES_REQUESTED** [review]

File: ghost/staff-service/lib/email-templates/partials/commonStyles.hbs (hunk 2 above)
Comment (lenabaidakova) [reply ghost/staff-service/lib/email-templates/partials/commonStyles.hbs]: Fixed :)

**Review by rshbhgrg — APPROVED** [review]
LGTM 🎉

**Pull request merged**